3_javascript/javascript1/week1/house-price-estimator/housePriceEstimator.js

```diff
@@ -0,0 +1,18 @@
+function estimatePrice(){
```

Comment:
again, looks good, I think you have not touched DOM Elements objects yet in classes, and you are using it already ⭐